kolibri/core/assets/src/views/CoreBase/index.vue

```diff
@@ -13,6 +13,7 @@
       :alwaysVisible="fixedAppBar"
       :mainWrapperScrollHeight="mainWrapperScrollHeight"
       :isHidden.sync="headerIsHidden"
+      :skipNextUpdate.sync="headerSkipNextUpdate"
```

Comment:
I'd like to move this into ScrollingHeader as part of the `isHidden` behavior, that it's not necessary to set it externally, but I haven't gotten that quite worked out yet.